klibs.md

```diff
@@ -3,13 +3,14 @@ Klibs
 
 Klibs are plugins which provide extra functionality to unikernels.
 
-As of Nanos [b66be2b](https://github.com/nanovms/nanos/commit/b66be2b642069519a14ea321d9bf5843921c08ab) there are 13 klibs in the kernel source tree:
+As of Nanos [9dc963b](https://github.com/nanovms/nanos/commit/9dc963bb2773b7a3dcb39aade3556350babae16b) there are 19 klibs in the kernel source tree:
```

Comment:
The klib count is actually 16 (those under the test folder are only for unit-testing the kernel code and shouldn't be counted, they are not even included in kernel releases).